package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "typescript-action",
+  "version": "0.0.0",
+  "private": true,
+  "description": "TypeScript template action",
+  "main": "lib/main.js",
+  "scripts": {
+    "build": "tsc",
+    "format": "prettier --write '**/*.ts'",
+    "format-check": "prettier --check '**/*.ts'",
+    "lint": "eslint src/**/*.ts",
+    "package": "ncc build --source-map --license licenses.txt",
+    "test": "jest",
+    "all": "npm run build && npm run format && npm run lint && npm run package && npm test"
```

Comment:
why is there a yarn.lock if we use npm here ?

Reply:
Because I forgot to change it. GitHub was using npm but I've been changing everything to use yarn instead because that's what is most commonly used at Scality.